_episodes/02-installation.md

```diff
@@ -22,61 +22,57 @@ be found in the
 We will first install Conda, and then ESMValTool. We end this chapter by testing
 that the installation was successful.
 
-## Install Conda
+## Install ESMValTool on Windows
```

Comment:
Maybe we could make a call-out box out of this?

Reply:
good idea, done.